### PR TITLE
Fixes #204 Update snippets for RE 2.0

### LINKED ISSUE #204: Update snippets for RE 2.0

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: snippets/qualification.code-snippets [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "prefix": "bs",
     "body": [
       "{",
-      "\"\\$schema\": \"https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/OSP_Qualification_Plan_Schema.json\",",
+      "\"\\$schema\": \"https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v3.0/OSP_Qualification_Plan_Schema.json\",",
```
Comment (Yuri05) [inline snippets/qualification.code-snippets:8]: I will create release 3.0 as soon as this PR was merged
(until then, schema link is invalid)

**Comment by Yuri05** [comment]
@msevestre Please review if you have time

**Comment by msevestre** [comment]
reviewing

File: snippets/qualification.code-snippets [hunk 2]
```diff
@@ -102,7 +102,7 @@
     "prefix": "as",
     "body": [
       "{",
-      "\"Id\":  ${1:<Enter Section Id>},",
+      "\"Reference\":  \"${1:<Enter Section Id>}\",",
```
Comment (msevestre) [inline snippets/qualification.code-snippets:105]: Id => Reference

**Review by msevestre — CHANGES_REQUESTED** [review]
One little chnage. After that great

File: snippets/qualification.code-snippets (hunk 2 above)
Comment (Yuri05) [reply snippets/qualification.code-snippets]: fixed

**Pull request merged**